x-pack/plugins/enterprise_search/public/applications/app_search/components/result_settings/result_settings_table/result_settings_text_fields_header.tsx

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { EuiTableHeader, EuiTableHeaderCell, EuiToolTip, EuiIcon } from '@elastic/eui';
+
+import { i18n } from '@kbn/i18n';
+
+export const ResultSettingsTextFieldsHeader: React.FC = () => {
+  return (
+    <>
+      <EuiTableHeader className="resultSettingsTable__columnLabels">
+        <EuiTableHeaderCell align="left" />
+        <EuiTableHeaderCell align="center" colSpan={2}>
+          {i18n.translate('xpack.enterpriseSearch.appSearch.engine.resultSettings.table.rawTitle', {
+            defaultMessage: 'Raw',
+          })}
+          <EuiToolTip
+            position="top"
+            content={i18n.translate(
+              'xpack.enterpriseSearch.appSearch.engine.resultSettings.table.rawDescription',
```

Comment:
I think technically this should be a `Tooltip` i18n type per https://github.com/elastic/kibana/blob/master/packages/kbn-i18n/GUIDELINE.md#message-types

```suggestion
              'xpack.enterpriseSearch.appSearch.engine.resultSettings.table.rawTooltip',
```